### PR TITLE
rename conf files to .example

### PR DESCRIPTION
**Pull request by aaperis**

### CODE REVIEW

**Review by jbygdell — APPROVED** [review]

File: docker-compose-htsget.yml [hunk 1]
```diff
@@ -14,14 +14,14 @@ services:
...
-      # of the file and the relevant configuration from the config-htsget/config.json 
+      # of the file and the relevant configuration from the config-htsget/examples/config.json
       # file, which are the lines for serverCert and serverKey.
       # The TLS version assumes that this compose file is running along with the
       # storage and interfaces.
       # If you want to run the htsget with TLS but without the storage and interfaces,
       # add the certificates in the volume below
       - starter-kit-storage-and-interfaces_shared:/shared
-      - ./config-htsget:/config
+      - ./config-htsget/examples:/config
```
Comment (andkaha) [inline docker-compose-htsget.yml]: It seems strange that the compose file refers to the `config-htsget/examples` subdirectory and not to files in `config-htsget`. I thought placing the example configurations in a `examples` subdirectory meant the _real_, _live_, configuration would be read from the parent directory. I.e., the user should copy the example files out of the `example` directory and then modify those copies.

The other common way to do this is to keep the example configurations in files with a `.example` filename suffix, and let the user copy these to the real configuration file, i.e. copy `config.json.example` to `config.json` and then modify that file.

This way, the deployed configuration is done in files that are not tracked by Git.

**Review by andkaha — CHANGES_REQUESTED** [review]
See comment.

File: docker-compose-htsget.yml (hunk 1 above)
Comment (nanjiangshu) [reply docker-compose-htsget.yml]: I agree with what @kusalananda suggested.

**Review by nanjiangshu — COMMENTED** [review]
Looks good in general! I think the comment Andreas suggested is valid and better to be addressed. Also, when the issue is solved, I think the changes in the GDI VM should not be git tracked. So far, we still see 

```
        modified:   config-htsget/config.json
        modified:   docker-compose-htsget.yml
```
on the GDI server for the folder `/home/ubuntu/starter-kit-htsget`

That should apply for all other cloned repos on the GDI server.

File: docker-compose-htsget.yml (hunk 1 above)
Comment (aaperis) [reply docker-compose-htsget.yml]: done

**Comment by aaperis** [comment]
> Looks good in general! I think the comment Andreas suggested is valid and better to be addressed. Also, when the issue is solved, I think the changes in the GDI VM should not be git tracked. So far, we still see
> 
> ```
>         modified:   config-htsget/config.json
>         modified:   docker-compose-htsget.yml
> ```
> 
> on the GDI server for the folder `/home/ubuntu/starter-kit-htsget`
> 
> That should apply for all other cloned repos on the GDI server.

The best way for this is to rename relevant files in the VM by adding a  `_prod` suffix. Changes underway.

**Review by andkaha — APPROVED** [review]
Loox ok now.

**Review by nanjiangshu — APPROVED** [review]

**Review by nanjiangshu — COMMENTED** [review]
Look good @aaperis ! However, 
there is one modification of the file `docker-compose-htsget.yml` on the GDI server for the repo `starter-kit-htsget`
```
-      - 3000:3000
+      - 8081:3000
```
Maybe we can change this line to 

```
      - "${PORT_HTSGET:-3000}:3000"
```
in `docker-compose-htsget.yml` and then add an `.env` with the content 

```
PROT_HTSGET=8081
``` on the GDI server.

The .env file should not be tracked by git. We may mention the way to set another port number for HTSGET in the README but I'm OK to skip it as well.

**Comment by aaperis** [comment]
@nanjiangshu There is no need to have here configuration settings from the VM. Files in the VM will be renamed in the VM and then will be untracked.

**Comment by andkaha** [comment]
I merged this with the understanding that the corresponding modifications to how example files are handled will be done in the other Starter Kit repository in a separate PR.

**Pull request merged**